packages/expo-camera/src/Camera.js

```diff
@@ -291,6 +284,16 @@ export default class Camera extends React.Component<PropsType> {
   }
 }
 
-export const Constants = Camera.Constants;
+function flipCameraStyle(shouldFlip) {
+  if (shouldFlip) {
+    return { transform: 'rotateY(180deg)' };
+  }
+  return { transform: 'none' };
+}
 
-const ExponentCamera = requireNativeViewManager('ExponentCamera', Camera);
+let ExponentCamera = null;
+if (Platform.OS !== 'web') {
+  ExponentCamera = requireNativeViewManager('ExponentCamera', Camera);
```

Comment:
I think we should explore adding an ExponentCamera module with a .web.ts version that exports a component that provides the native ExponentCamera component API and wraps the `<video>` element, instead of doing the `if (Platform.OS === 'web')` check in `render()` above. This way we can make sure  that all of the ExponentCamera props are implemented (ex: `onCameraReady` is missing).